module/submissions/helper.js

```diff
@@ -148,6 +148,11 @@ module.exports = class SubmissionsHelper {
 
                     // Push new submission to kafka for reporting/tracking.
                     this.pushInCompleteSubmissionForReporting(submissionDocument._id);
+
+                    if( submissionDocument.referenceFrom === messageConstants.common.PROJECT ) {
+                        this.pushSubmissionToImprovementService(submissionDocument);
```

Comment:
@aman-tunerlabs - Why not use pick here as well?

Reply:
@aks30 some other data also we required like assessors etc